definitions/video-datazoom_video/definition.yml

```diff
@@ -0,0 +1,29 @@
+domain: VIDEO
+# Represents the video player experience across all instances of a customer's Browser Video players instrumented with New Relic's Video SDKs
+type: DATAZOOM_VIDEO
+
+synthesis:
+  rules:
+    - identifier: configuration_name
+      name: configuration_name
+
+      encodeIdentifierInGUID: false
+
+      conditions:
+        - attribute: eventType
+          value: Datazoom
+        - attribute: player_name
+          present: true
+      
+      tags:
+        player_name:
```

Comment:
just for my learning, shouldn't this be the entity.name? Is configuration_name what the user expects as the name?

Reply:
In this case, Player Name is the underlying player technology - JW Player, AVPlayer, etc, whereas configuration name is more similar to appName, it's customer defined.

Reply:
perfect thanks!